L1Trigger/L1TMuonEndCap/src/PrimitiveSelection.cc

```diff
@@ -7,47 +7,49 @@
 #include "L1Trigger/L1TMuonEndCap/interface/EMTFGEMDetId.h"
 #include "L1Trigger/L1TMuonEndCap/interface/EMTFGEMDetIdImpl.h"
 
-
 #include "helper.h"  // merge_map_into_map, assert_no_abort
 
-#define NUM_CSC_CHAMBERS 6*9   // 18 in ME1; 9x3 in ME2,3,4; 9 from neighbor sector.
-                               // Arranged in FW as 6 stations, 9 chambers per station.
-#define NUM_RPC_CHAMBERS 7*8   // 6x2 in RE1,2; 12x2 in RE3,4; 6 from neighbor sector.
-                               // Arranged in FW as 7 stations, 6 chambers per station. (8 with iRPC)
-#define NUM_GEM_CHAMBERS 6*9   // 6 in GE1/1; 3 in GE2/1; 2 from neighbor sector.
-                               // Arranged in FW as 6 stations, 9 chambers per station, mimicking CSC. (unconfirmed!)
-
-
-void PrimitiveSelection::configure(
-      int verbose, int endcap, int sector, int bx,
-      int bxShiftCSC, int bxShiftRPC, int bxShiftGEM,
-      bool includeNeighbor, bool duplicateTheta,
-      bool bugME11Dupes
-) {
+#define NUM_CSC_CHAMBERS \
+  6 * 9  // 18 in ME1; 9x3 in ME2,3,4; 9 from neighbor sector. \
```

Comment:
@smuzaffar the compiler does not digest easily these multiline comments:
```
>> Compiling  /build/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_0_X_2019-05-28-2300/src/L1Trigger/L1TMuonEndCap/src/PhiMemoryImage.cc 
  /build/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_0_X_2019-05-28-2300/src/L1Trigger/L1TMuonEndCap/src/PrimitiveSelection.cc:13:10: warning: multi-line comment [-Wcomment]
    6 * 9  // 18 in ME1; 9x3 in ME2,3,4; 9 from neighbor sector. \
          ^
  /build/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_0_X_2019-05-28-2300/src/L1Trigger/L1TMuonEndCap/src/PrimitiveSelection.cc:16:10: warning: multi-line comment [-Wcomment]
    7 * 8  // 6x2 in RE1,2; 12x2 in RE3,4; 6 from neighbor sector. \
          ^
  /build/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_11_0_X_2019-05-28-2300/src/L1Trigger/L1TMuonEndCap/src/PrimitiveSelection.cc:19:10: warning: multi-line comment [-Wcomment]
    6 * 9  // 6 in GE1/1; 3 in GE2/1; 2 from neighbor sector. \
          ^
```
This needs to be fixed manually before moving forward with this PR

Reply:
fixed